server/src/test/java/org/opensearch/action/support/master/TransportMasterNodeActionTests.java

```diff
@@ -419,11 +419,11 @@ public void testDelegateToFailingMaster() throws ExecutionException, Interrupted
         boolean failsWithConnectTransportException = randomBoolean();
         boolean rejoinSameMaster = failsWithConnectTransportException && randomBoolean();
         Request request = new Request().masterNodeTimeout(TimeValue.timeValueSeconds(failsWithConnectTransportException ? 60 : 0));
```

Comment:
`masterNodeTimeout ` -> `clusterManagerNodeTimeout`

Reply:
I will rename `masterNodeTimeout` in the next phase, which is mentioned in issue https://github.com/opensearch-project/OpenSearch/issues/1684.